modules/ossm-installation-activities.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-installation.adoc
+
+[id="ossm-installation-activities_{context}"]
+= {ProductName} installation activities
+
+The {ProductName} installation process creates two different projects (namespaces):
+
+* istio-operator project (1 pod)
+* istio-system project (17 pods)
+
+You first create a Kubernetes _operator_. This operator defines and monitors a _custom resource_ that manages the deployment, updating, and deletion of the {ProductShortName} components.
+
+Depending on how you define the custom resource file, you can install one or more of the following components when you install the {ProductShortName}:
+
+* *Istio* - based on the  open source link:https://istio.io/[Istio] project, lets you connect, secure, control, and observe the microservices that make up your applications.
+* *Jaeger* - based on the open source link:https://www.jaegertracing.io/[Jaeger] project, lets you perform tracing to monitor and troubleshoot transactions in complex distributed systems.
+* *Kiali* - based on the open source link:https://www.kiali.io/[Kiali] project, Kiali provides observability for your service mesh. Using Kiali lets you view configurations, monitor traffic, and view and analyze traces in a single console.
+* *Launcher* - based on the open source link:http://fabric8.io/[fabric8] community, this integrated development platform helps you build cloud native applications and microservices. {ProductName} includes several boosters that let you explore features of the {ProductShortName}.
+
+During the installation the operator creates an Ansible job that runs an Ansible playbook that performs the following installation and configuration tasks automatically:
```

Comment:
The operator doesn't use ansible anymore and doesn't create the openshift-ansible-istio-installer-job or the istio-system namespace

Reply:
Removed.